index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Lab1_Purshak_Illa</title>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	</head>
+
+	<body>
+		<header>
+			<a href="index.html"><b>Web</b>Studio</a>
+			<nav>
+				<ul>
+					<li><a href="#">Студія</a></li>
+					<li><a href="#">Портфоліо</a></li>
+					<li><a href="#">Контакти</a></li>
+				</ul>
+			</nav>
+			<ul>
+				<li><a href="mailto:info@devstrudio.com">info@devstrudio.com</a></li>
+				<li><a href="tel:+380994814090">+38 099 481 40 90</a></li>
+			</ul>
+		</header>
+
+		<main>
+			<!-- 4 секції замість div -->
+			<section>
+				<h1>ЕФЕКТИВНІ РІШЕННЯ ДЛЯ<br />ВАШОГО БІЗНЕСУ</h1>
+				<button type="button">Замовити послугу</button>
+			</section>
+
+			<section>
+				<!-- Правильно розташувати блоки -->
+				<ul>
+					<li>
+						<h3>УВАГА ДО ДЕТАЛЕЙ</h3>
+						<p>
+							Ідейні міркування, і навіть початок повсякденної роботи з
+							формування позиції.
+						</p>
+					</li>
+
+					<li>
+						<h3>ПУНКТУАЛЬНІСТЬ</h3>
+						<p>
+							Завдання організації, особливо рамки і місце навчання кадрів тягне
+							у себе.
+						</p>
+					</li>
+
+					<li>
+						<h3>ПЛАНУВАННЯ</h3>
+						<p>
+							Так само консультація з широким активом значною мірою зумовлює.
+						</p>
+					</li>
+
+					<li>
+						<h3>СУЧАСНІ ТЕХНОЛОГІЇ</h3>
+						<p>
+							Значимість цих проблем настільки очевидна, що реалізація планових
+							завдань.
+						</p>
+					</li>
+				</ul>
+			</section>
+
+			<section>
+				<h2>Чим ми займаємося</h2>
+				<ul>
+					<li>
+						<img src="images/tablet.jpg" alt="Чоловік програмує з планшетом" />
+					</li>
+					<li><img src="images/phone.jpg" alt="Чоловік з телефоном" /></li>
+					<li>
+						<img
```

Comment:
Always specify both the height and width attributes for images. If height and width are set, the space required for the image is reserved when the page is loaded. However, without these attributes, the browser does not know the size of the image, and cannot reserve the appropriate space to it.